src/openrct2/ride/TrackData.h

```diff
@@ -90,6 +90,9 @@ struct TrackElementDescriptor
 
     rct_trackdefinition Definition;
     uint8_t SpinFunction;
+
+    std::function<int32_t(const int16_t)> VerticalFactor;
```

Comment:
I would prefer we have a well defined function signature here. e.x.:
```
using TrackComputeFunction = int32_t(*)(const int16_t);
```
instead of std::function.